src/ctree-segments/ctree-segment-behavior.html

```diff
@@ -30,6 +30,7 @@
 <link rel="import" href="../../bower_components/polymer/polymer.html">
 <link rel="import" href="../../bower_components/polymer/lib/utils/async.html">
 
+
```

Comment:
Another consistency point that we don't usually have 2 empty lines after the imports.